fava_investor/modules/performance/balances.py

```diff
@@ -4,12 +4,12 @@
 def get_closed_tree_with_value_accounts_only(accapi, config):
     tree = accapi.ledger.root_tree_closed
     accounts_to_keep = get_value_accounts_and_parents(tree, accapi.ledger.accounts,
-            config.get("accounts_pattern", "^Assets:.*"))
+            config.get("accounts_patterns", ["^Assets:.*"]))
```

Comment:
I made this change because a list seems superfluous. Is there a case that the list handles that a pattern cannot?

Reply:
right, doesnt seem necessary